tasks/steps/deploy.yml

```diff
@@ -2,6 +2,9 @@
 ---
 - name: "StackHead::Container || Build src folder list || Project: {{ project_name }}"
   block:
+    - name: "Container hooks location"
+      set_fact:
+        managedContainerVolumePaths: ["{{ stackhead__containerhooks_location }}"]
```

Comment:
Rename managedContainerVolumePaths fact as it does not only contain volume paths anymore